app/src/pages/playground/PlaygroundDatasetExamplesTable.tsx

```diff
@@ -116,9 +129,47 @@ const updateExampleResponsesMap = ({
   currentMap,
 }: {
   instanceId: number;
-  response: ChatCompletionResult | TextChunk | ToolCallChunk;
+  response:
+    | ChatCompletionResult
+    | TextChunk
+    | ToolCallChunk
+    | ChatCompletionOverDatasetMutationPayload;
   currentMap: InstanceToExampleResponsesMap;
 }): InstanceToExampleResponsesMap => {
+  if (response.__typename === "ChatCompletionOverDatasetMutationPayload") {
+    const newMap = { ...currentMap };
+    for (const example of response.examples) {
+      const { datasetExampleId, result } = example;
+      switch (result.__typename) {
+        case "ChatCompletionMutationError": {
+          // TODO: handle error
```

Comment:
This PR or future PR?

Reply:
for future because this diverges from subscription which has no error handling at the example level: it's all or nothing

Reply:
ahh yeah this is fixed in my pr let me merge shortly and can rebase!

Reply:
ok sounds good. thanks!